[Bug]: building on FreeBSD fails because of network-viewer.plugin

### Bug description

[Reported on Discord](https://discord.com/channels/847502280503590932/1202695058738909294/1202695058738909294)

```bash
collectors/network-viewer.plugin/network-viewer.c:31:13: error: use of undeclared identifier 'TCP_FIN_WAIT2'
```


### Expected behavior

The network-viewer plugin should be built only on Linux.

### Steps to reproduce

Install (kickstart.sh) Netdata on FreeBSD.


### Installation method

kickstart.sh

### System info

```shell
FreeBSD
```


### Netdata build info

```shell
netdata-v1.44.0-273
```


### Additional info

_No response_

--- a/packaging/installer/functions.sh
+++ b/packaging/installer/functions.sh
@@ -270,6 +270,7 @@ prepare_cmake_options() {
   enable_feature PLUGIN_SLABINFO "${IS_LINUX}"
   enable_feature PLUGIN_CGROUP_NETWORK "${IS_LINUX}"
   enable_feature PLUGIN_LOCAL_LISTENERS "${IS_LINUX}"
+  enable_feature PLUGIN_NETWORK_VIEWER "${IS_LINUX}"
   enable_feature PLUGIN_EBPF "${ENABLE_EBPF:-0}"
   enable_feature PLUGIN_LOGS_MANAGEMENT "${ENABLE_LOGS_MANAGEMENT:-0}"
   enable_feature LOGS_MANAGEMENT_TESTS "${ENABLE_LOGS_MANAGEMENT_TESTS:-0}"

